### PR TITLE
contributing + governance updates

### PR DESCRIPTION
**Pull request by leodido**
This PR is **WIP** because it needs to be presented (and discussed) during the Falco community call.

This PR refines the CONTRIBUTING.md and GOVERNANCE.md (organization-wide) clarifying topics like:
- @poiana resources
- contributor of the month
- maintainership status
- conflict resolution and voting

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -107,60 +133,72 @@ Once your reviewer is happy, they will say `/lgtm` which will apply the
 Your PR will be automatically merged once it has the `lgtm` and `approved`
 labels, does not have any `do-not-merge/*` labels, and all status checks (eg., rebase, tests, DCO) are positive.
 
+### Reviews
+
+Reviewing a pull request is also a very good way of contributing
```
Comment (leogr) [inline CONTRIBUTING.md]:
```suggestion
Reviewing a pull request is also a very good way of contributing.

```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -1,55 +1,105 @@
...
+  - [Maintainer responsibilities](#maintainer-responsibilities)
+  - [When does a maintainer lose maintainer status](#when-does-a-maintainer-lose-maintainer-status)
+    - [How to step down](#how-to-step-down)
+    - [Project inactivity](#project-inactivity)
+  - [Conflict resolution and voting](#conflict-resolution-and-voting)
+  - [Adding new projects to the falcosecurity GitHub organization](#adding-new-projects-to-the-falcosecurity-github-organization)
+
+## Maintainers
+
+The list of the current [maintainers](./maintainers.yaml) is automatically updated by the Falco Infra.
```
Comment (leogr) [inline GOVERNANCE.md]:
```suggestion
The list of the current [maintainers](./maintainers.yaml) is automatically updated by the [Falco Infra](https://github.com/falcosecurity/test-infra).
```

File: GOVERNANCE.md [hunk 3]
```diff
@@ -1,55 +1,105 @@
...
+
+Maintainers of [offically supported projects](https://github.com/falcosecurity/evolution#official-support) can use a double vote in organization-wide issues or when in deadlocks.
+
+In case the conflict is about changing this governance or on maintainership status it is required a majority of 66% of votes to resolve the disputes.
+
+In all the other cases, the voting process is a simple majority in which each maintainer receives one vote.
+
+## Adding new projects to the falcosecurity GitHub organization
+
+New projects will be added to the falcosecurity organization via GitHub issue discussion in the `falcosecurity/evolution` repository.
```
Comment (leogr) [inline GOVERNANCE.md]:
```suggestion
New projects will be added to the falcosecurity organization via GitHub issue discussion in the [falcosecurity/evolution](https://github.com/falcosecurity/evolution) repository.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,35 +2,61 @@
...
 
-[Wednesdays at 8am Pacific](https://lists.cncf.io/g/cncf-falco-dev/calendar) on [Zoom](https://zoom.us/my/cncffalcoproject).
+[Wednesdays at 4PM UTC](https://lists.cncf.io/g/cncf-falco-dev/calendar) on [Zoom](https://zoom.us/my/cncffalcoproject).
 
-## Issues
+You can subscribe to the calendar with GNOME Calendar, iCal, Google Calendar, or your favorite application using this ICS feed:
+
+```console
+https://lists.cncf.io/g/cncf-falco-dev/ics/4188901/1750099167/feed.ics
+```
```
Comment (leogr) [inline CONTRIBUTING.md:42]: It seems to be redundant with https://github.com/falcosecurity/community#falco-community-calls

Would we remove redundant info from https://github.com/falcosecurity/community#falco-community-calls ?

**Review by leogr — COMMENTED** [review]
Great improvements!

I left some very minor comments, but overall I already agree with the updates :+1: 

Thank you!

File: CONTRIBUTING.md (hunk 4 above)
Comment (leodido) [reply CONTRIBUTING.md:42]: I prefer this info to be redundant, tbh.

So that the users can find it in many places.

**Comment by leodido** [comment]
@leogr I've accepted (most of) your suggestions!

**Review by leogr — APPROVED** [review]

**Pull request merged**